R/pkg/R/schema.R

```diff
@@ -69,10 +69,11 @@ structType.structField <- function(x, ...) {
 #' @param ... further arguments passed to or from other methods
 print.structType <- function(x, ...) {
   cat("StructType\n",
-      sapply(x$fields(), function(field) { paste("|-", "name = \"", field$name(),
-                                           "\", type = \"", field$dataType.toString(),
-                                           "\", nullable = ", field$nullable(), "\n",
-                                           sep = "") })
+      sapply(x$fields(), function(field) {
+             paste("|-", "name = \"", field$name(),
+                   "\", type = \"", field$dataType.toString(),
+                   "\", nullable = ", field$nullable(), "\n",
+                   sep = "") })
```

Comment:
This is what I mean.
https://github.com/sarutak/spark/blob/SPARK-9052/R/pkg/R/schema.R#L78